### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by bendavies**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/WebBundle/Resources/translations/messages.th.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@ sylius:
         no_shipping_charges: ไม่มีค่าธรรมเนียมการจัดส่ง
         no_taxes: ไม่รวมภาษี
         state:
-            order_confimed: ยืนยันการสั่งซื้อ
+            order_confirmed: ยืนยันการสั่งซื้อ
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/WebBundle/Resources/translations/messages.th.yml]: We should only modify ``en`` translation files.

Comment (bendavies) [inline src/Sylius/Bundle/WebBundle/Resources/translations/messages.th.yml]: ah, you have an auto translation process, right?

Comment (pamil) [inline src/Sylius/Bundle/WebBundle/Resources/translations/messages.th.yml]: CrowdIn should work that out itself :)

**Comment by pamil** [comment]
Nice! 👍

File: src/Sylius/Bundle/WebBundle/Resources/translations/messages.th.yml (hunk 1 above)
Comment (bendavies) [inline src/Sylius/Bundle/WebBundle/Resources/translations/messages.th.yml]: done

**Comment by pjedrzejewski** [comment]
Thanks Ben!

**Pull request merged**